app/web/next-env.d.ts

```diff
@@ -1,5 +1,6 @@
 /// <reference types="next" />
 /// <reference types="next/image-types/global" />
+/// <reference path="./.next/types/routes.d.ts" />
```

Comment:
I did not upgrade this, it got changed when I ran `yarn build` so I think it's okay.